README.md

```diff
@@ -1,16 +1,24 @@
 # C++/CUDA Extensions in PyTorch
 
-An example of writing a C++ extension for PyTorch. See
+An example of writing a C++/CUDA extension for PyTorch. See
 [here](http://pytorch.org/tutorials/advanced/cpp_extension.html) for the accompanying tutorial.
+This repo demonstrates how to write an example `extension_cpp.ops.lltm`
+custom op that has both custom CPU and CUDA kernels.
 
-There are a few "sights" you can metaphorically visit in this repository:
+To build:
+```
+pip install .
```

Comment:
Should we do -e to have a "develop" equivalent?

Reply:
Yup